src/features/flows/thingy91x/index.ts

```diff
@@ -0,0 +1,315 @@
+        documentation: {
+            label: 'nRF Cloud multi-service',
+            href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/cellular/nrf_cloud_multi_service/README.html',
+        },
+        firmwareNote: {
+            title: 'Increased power consumption',
+            content:
+                'Modem Trace is enabled; the current consumption will be higher than usual.',
+        },
+        firmware: [
+            {
+                core: 'Application',
+                file: '___FILL_INN___.hex',
+                link: {
+                    label: 'nRF Cloud multi-service',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/cellular/nrf_cloud_multi_service/README.html',
+                },
+            },
+        ],
+    },
+    {
+        name: 'Shell Command Line Interface',
+        type: 'buttonless-dfu',
+        description: 'Evaluate throughput, connectivity, and more.',
+        documentation: {
+            label: 'Modem Shell',
+            href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/cellular/modem_shell/README.html',
+        },
+        firmwareNote: {
+            title: 'Increased power consumption',
+            content:
+                'Modem Trace is enabled; the current consumption will be higher than usual.',
+        },
+        firmware: [
+            {
+                core: 'Application',
+                file: '___FILL_INN___.hex',
+                link: {
+                    label: 'Modem Shell',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/cellular/modem_shell/README.html',
+                },
+            },
+        ],
+    },
+] as Choice[];
+
+const verifyConfig = {
+    settings: [
+        {
+            ref: 'Prototyping explorer',
+            vComIndex: 0,
+            mode: 'SHELL' as const,
+        },
+        {
+            ref: 'AT Commands',
+            vComIndex: 0,
+            mode: 'LINE' as const,
+        },
+        {
+            ref: 'nRF Cloud multi-service',
+            vComIndex: 0,
+            mode: 'SHELL' as const,
+        },
+        {
+            ref: 'Shell Command Line Interface',
+            vComIndex: 0,
+            mode: 'SHELL' as const,
+        },
+    ],
+    commands: [
+        {
+            title: 'Manufacturer',
+            command: 'AT+CGMI',
+            responseRegex: '(.*)',
+        },
+        {
+            title: 'Hardware version',
+            command: 'AT%HWVERSION',
+            responseRegex: '%HWVERSION: (.*)',
+        },
+        {
+            title: 'International Mobile Equipment Identity',
+            command: 'AT+CGSN=1',
+            responseRegex: '\\+CGSN: "(.*)"',
+            copiable: true,
+        },
+    ],
+};
+
+const evaluateConfig = [
+    {
+        ref: 'Prototyping explorer',
+        resources: [
+            {
+                title: 'Explore the possibilities',
```

Comment:
This is way more generic than other entries. Also, other entries don't have the `title`. Why is that?

Reply:
This is the title on the button for this option. All other options are to open apps and show a standard "open x-app" text.

Reply:
Isn't the line 174 the title of the button?

Reply:
You're right, sorry. But the explanation is the same.
The options that open apps have a standard "app" title, which is why this option isn't used there.